onos-operator/charts/topo-operator/crds/relation.yaml

```diff
@@ -41,18 +44,18 @@ spec:
                 source:
                   type: object
                   required:
-                    - name
+                    - uri
                   properties:
-                    name:
```

Comment:
I think we should leave this field here and mark it `deprecated: true` for now. We need to get used to making API changes without breaking things. Otherwise, even small changes like this seem to become really disruptive during the period of inconsistency between different components

Reply:
Good idea. I put it back and marked as deprecated.